applications/nrf5340_audio/src/utils/board_init.c

```diff
@@ -103,6 +106,7 @@ int nrf5340_audio_dk_init(void)
 		return ret;
 	}
 
+#if defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP)
```

Comment:
Can we do a normal `if (IS_DEFINED(...)` here?

Reply:
Yes good point, will change.